control-plane/pkg/apis/internals/kafka/eventing/data_plane.go

```diff
@@ -27,6 +27,11 @@ const (
 	ConfigMapVolumeName = "kafka-resources"
 
 	DispatcherVolumeName = "contract-resources"
+
+	DataPlanePodKindLabelKey    = "app.kubernetes.io/kind"
+	DispatcherPodKindLabelValue = "kafka-dispatcher"
```

Comment:
I don't see this label on the dispatcher pods. E.g.: https://github.com/knative-extensions/eventing-kafka-broker/blob/c9e23202d9af3748fe650886569e135b13c7c6d4/data-plane/config/broker/500-dispatcher.yaml#L22-L26. What am I missing?

Reply:
Pod label, not statefulset label

https://github.com/knative-extensions/eventing-kafka-broker/blob/c9e23202d9af3748fe650886569e135b13c7c6d4/data-plane/config/broker/500-dispatcher.yaml#L41

Reply:
ah. ok :+1:

Reply:
Thanks for clarifying